IFC4x3/Templates/Partial Templates/Material Definition/Material/DocTemplateDefinition.xml

```diff
@@ -16,6 +16,34 @@
 				<DocModelRuleEntity Name="IfcLabel" />
 			</Rules>
 		</DocModelRuleAttribute>
+		<DocModelRuleAttribute Name="HasProperties" Identification="HasProperties">
+			<Rules>
+				<DocModelRuleEntity Name="IfcMaterialProperties">
+					<Rules>
+						<DocModelRuleAttribute Name="Name" Identification="PsetName">
+							<Rules>
+								<DocModelRuleEntity Name="IfcIdentifier" />
+							</Rules>
+						</DocModelRuleAttribute>
+						<DocModelRuleAttribute Name="Description">
+							<Rules>
+								<DocModelRuleEntity Name="IfcText" />
+							</Rules>
+						</DocModelRuleAttribute>
+						<DocModelRuleAttribute Name="Properties" Identification="PsetProperties">
+							<Rules>
+								<DocModelRuleEntity Name="IfcPropertySingleValue" />
+								<DocModelRuleEntity Name="IfcPropertyBoundedValue" />
+								<DocModelRuleEntity Name="IfcPropertyEnumeratedValue" />
+								<DocModelRuleEntity Name="IfcPropertyListValue" />
+								<DocModelRuleEntity Name="IfcPropertyTableValue" />
+								<DocModelRuleEntity Name="IfcPropertyReferenceValue" />
```

Comment:
Are we intentionally not including `IfcComplexProperty`?

Reply:
Discussion point. I didn't include since it is not included in the official 4.3 documentation. Btw, neither was reference value (but I included that one).